src/containers/shared/components/Transaction/DefaultSimple.tsx

```diff
@@ -0,0 +1,129 @@
+import { isValidClassicAddress } from 'ripple-address-codec'
+import { formatAmount } from '../../../../rippled/lib/txSummary/formatAmount'
+import { Account } from '../Account'
+import { Amount } from '../Amount'
+import Currency from '../Currency'
+import { SimpleGroup } from './SimpleGroup'
+import { SimpleRow } from './SimpleRow'
+import { TransactionSimpleProps } from './types'
+
+const DEFAULT_TX_ELEMENTS = [
+  'Account',
+  'Fee',
+  'Flags',
+  'LastLedgerSequence',
+  'NetworkID',
+  'Sequence',
+  'SigningPubKey',
+  'TransactionType',
+  'TxnSignature',
+  'ctid',
+  'date',
+  'warnings',
+]
+
+const processValue = (value: any) => {
+  if (typeof value === 'string') {
+    if (isValidClassicAddress(value)) {
+      return <Account account={value} />
+    }
+    if (value.length > 300) {
+      return `${value.substring(0, 300)}...`
+    }
+    return value
+  }
+
+  return JSON.stringify(value)
+}
+
+const getRow = (
+  key: any,
+  value: any,
+  depth: number = 0,
+  uniqueKey: string = '',
+) => {
+  if (Array.isArray(value) && depth < 1) {
+    return (
+      <SimpleGroup key={key} title="" data-test={key}>
+        <>
+          {value.map((childValue, index) => {
+            if (Object.keys(childValue).length === 1) {
+              const childKey = Object.keys(childValue)[0]
+              return getRow(
+                childKey,
+                childValue[childKey],
+                depth,
+                index.toString(),
+              )
+            }
+            return getRow(index.toString(), childValue, depth, index.toString())
+          })}
+        </>
+      </SimpleGroup>
+    )
+  }
+
+  if (key === 'Amount') {
```

Comment:
Would this this fall under the case on line 87?

Reply:
No - this catches the case where `Amount` represents XRP